source/draw/gpu/opengl/OpenGLContext.ooc

```diff
@@ -66,6 +71,8 @@ OpenGLContext: class extends GpuContext {
 	}
 	init: func ~window (display: Pointer, nativeBackend: Long) { this init(GLContext createContext(display, nativeBackend)) }
 	free: override func {
+		if (this _defaultFontGpu != null)
+			this _defaultFontGpu free()
```

Comment:
Freeing textures can't be done after freeing the recycle bin but super has to be at the end of the destructor.